src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -117,6 +119,26 @@ class GrabbitConfiguration {
         )
     }
 
+    /**
+     * Removes '/.' from {@code pathConfigurations}'s path if exists in the end, and Sorts the {@code pathConfigurations}
+     * on the basis of paths' depth
+     *
+     * @param pathConfigurations
+     *          Collection of {@link PathConfiguration} to sort
+     * @return
+     *          Sorted Collection of {@link PathConfiguration}
+     */
+    private static Collection<PathConfiguration> getSortedPathConfigurationsByDepth(
```

Comment:
Reason I had it as 
```groovy
List<PathConfiguration>
```
 was that List implies order, which is evident of a sorted Collection of items. 

Would need to update the JavaDoc above as well.